app/controllers/kudos_controller.rb

```diff
@@ -72,4 +77,15 @@ def set_kudo
   def kudo_params
     params.require(:kudo).permit(:title, :content, :giver_id, :receiver_id, :company_value_id)
   end
+
+  # customize Pundit user
+  def pundit_user
```

Comment:
I would extract that to application controller :)

Reply:
make sense, as we are using the same "user model" in whole application :D